### PR TITLE
Fix an issue when passphrase is cancelled

### PR DESCRIPTION
**Pull request by GuillaumeSmaha**
Cancel the window asking passphrase crash the app due a missing error.

### CODE REVIEW

File: k9mail/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1122,6 +1122,7 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="openpgp_error">OpenPGP Error: %s</string>
     <string name="openpgp_unknown_error">Unknown OpenPGP Error</string>
     <string name="openpgp_user_id">User Id</string>
+    <string name="openpgp_cancel_by_user">Cancel by user</string>
```
Comment (maniac103) [inline k9mail/src/main/res/values/strings.xml]: Grammar issue here: this should be 'Cancelled by user'.

Comment (GuillaumeSmaha) [inline k9mail/src/main/res/values/strings.xml]: Fixed

Comment (GuillaumeSmaha) [inline k9mail/src/main/res/values/strings.xml]: Should I rename "openpgp_cancel_by_user" too ?

**Comment by Diagoras** [comment]
+1

**Pull request closed without merging**